challenge1/styles.css

```diff
@@ -15,14 +14,114 @@ html {
     font-family: "Ubuntu", sans-serif;
 }
 
+header {
+    display: flex;
+    justify-content:space-between;
+    align-items: center;
+    padding: 1rem 2rem;
+    gap:2rem
+}
+.filters {
+    display: flex;
+    flex-direction: row;
+    gap: 1rem;
+}
+.title {
+    padding: 0 2rem 0 2rem ;
+}
+
+.search-container {
+    position: relative;
+    display: flex;
+    align-items: center;
+}
+.search-container ::placeholder {
+    font-family: "Manrope", sans-serif;
+    font-weight: 500;
+    font-size: 1.6rem;
+    color: #ABAEB7;
+}
+
+#searchInput {
+    padding: 1.2rem 1.6rem 1.2rem 1.6rem;
+    font-size: 1.6rem;
+    border-color: #ABAEB7;
```

Comment:
We should move this color to a variable so we don't repeat it (and the same for the other colors)